### PR TITLE
Fixed your shit

### PR DESCRIPTION
**Pull request by lukasvdberk**

### CODE REVIEW

**Comment by lukasvdberk** [comment]
pls papa

**Comment by lukasvdberk** [comment]
alt

**Comment by lukasvdberk** [comment]
pls merge

**Comment by lukasvdberk** [comment]
I fixed for free

**Comment by lukasvdberk** [comment]
I want to feed my family

**Comment by lukasvdberk** [comment]
papa

**Comment by lukasvdberk** [comment]
merge pls

**Review by sonofice — APPROVED** [review]
finally merged that shit zoon

File: main.py [hunk 1]
```diff
@@ -99,4 +110,4 @@ def main():
 # Start of the script/game
 if __name__ == "__main__":
     create_cards()
-    main()
\ No newline at end of file
+    main()
```
Comment (abcdan) [inline main.py:113]: Good change!

File: main.py [hunk 2]
```diff
@@ -40,11 +40,22 @@ def shuffle():
     else:
         return
 
+def get_player_count_input():
+    while True:
+        try:
+            player_count = input("How many players are participating?: ")
```
Comment (abcdan) [inline main.py:46]: Smart chang e!

**Pull request merged**